.github/workflows/ci-app_block.yml

```diff
@@ -1,6 +1,16 @@
 name: Block app edits
 
-on: ["pull_request"]
+on:  # Trigger the workflow on push or pull request, but only for the master branch
+  pull_request:
+    branches: [master, "release/*"]
+    types: [opened, reopened, ready_for_review, synchronize]
+    paths-ignore:
+      - "src/lightning_app/**"  # todo: implement job skip
+      - "tests/tests_app/**"  # todo: implement job skip
+      - "tests/tests_app_examples/**"  # todo: implement job skip
+      - "examples/app_*"  # todo: implement job skip
+      - "docs/source-app/**"  # todo: implement job skip
```

Comment:
any reason for this?

Reply:
@Borda It is called `ci-app_block.yml`. It shouldn't trigger if changes are made to app no?

Reply:
Reverted the change.